### PR TITLE
Add ga events for travel review and travel success

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Adds GA events for the review page and check in travel submit success
- if setECheckinStartedCalled is null or false then label should include the string 45MR
- use adswerve extension on chrome to inspect

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#72961](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72961)

## Testing done

- Visual

## Screenshots

Screenshot 2024-01-05 at 4 55 59 PM
Screenshot 2024-01-05 at 4 56 46 PM
Screenshot 2024-01-05 at 4 58 08 PM

## What areas of the site does it impact?

check-in -> day-of -> travel review and complete pages

## Acceptance criteria

- [ ] Events fire and are labeled properly based on setECheckinStartedCalled value

### CODE REVIEW

File: src/applications/check-in/day-of/pages/TravelReview.jsx [hunk 1]
```diff
@@ -28,12 +35,26 @@ const TravelQuestion = props => {
   };
   const validation = () => {
     if (agree) {
+      recordEvent({
```
Comment (brianseek) [inline src/applications/check-in/day-of/pages/TravelReview.jsx:38]: these events are already firing in the TravelPage component. Adding here adds duplicate calls.

File: src/applications/check-in/hooks/useSendTravelPayClaim.jsx [hunk 2]
```diff
@@ -45,13 +51,22 @@ const useSendTravelPayClaim = appointment => {
           setTravelPayClaimError(true);
         })
         .finally(() => {
+          recordEvent({
```
Comment (brianseek) [inline src/applications/check-in/hooks/useSendTravelPayClaim.jsx:54]: this should be done on the API event and not add an additional nav event

**Review by brianseek — APPROVED** [review]
Somethings to clean up on this one. The review page was already working as intended and this added a double event. On the API call, there are already api events being called, we should be adding the 45MR there instead of adding a new nav event.

**Review by brianseek — CHANGES_REQUESTED** [review]
I accidentally hit the approve option on the other review :)

**Comment by benbrasso-agile6** [comment]
Would there have been a reason the events on the buttons on the Review page weren't working? I can try to look for them again, but wasn't seeing anything firing on those buttons.

File: src/applications/check-in/day-of/pages/TravelReview.jsx (hunk 1 above)
Comment (wullaski) [reply src/applications/check-in/day-of/pages/TravelReview.jsx:38]: Thanks good catch!

**Comment by wullaski** [comment]
@benbrasso-agile6 The events are firing on the review page for me and the API call for travel pay success. Are you looking specifically for the 45MR string? That probably won't show up if you are testing on stage. It was showing at one point because I had the logic backwards.

**Comment by benbrasso-agile6** [comment]
I left a comment here - https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/72961

Pulled from production this morning

**Review by brianseek — APPROVED** [review]
Looks good. Tested locally with 45MR and without, all events were as they should be.

**Pull request merged**